crates/transaction-pool/src/lib.rs

```diff
@@ -419,6 +419,14 @@ where
         self.pool.best_transactions_with_base_fee(base_fee)
     }
 
+    fn best_transactions_with_attributes(
+        &self,
+        base_fee: u64,
+        blob_fee: Option<u64>,
```

Comment:
I want a new struct type for this, this way it's easier to call and it won't break if we need more attributes in the future

Reply:
I added `BestTransactionsAttributes` struct as the argument